### DISCUSSION TITLE
Error occured while attempting to load SSL for the production domain

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#835] [General]
Hi, 
We encountered an error when attempting to associate SSL with the production domain, as we are unable to load the key file.(screenshot attached)

![Screenshot (35)](https://github.com/Sunbird-cQube/community/assets/130358614/2b721843-9774-4c8e-b23c-c26647a7d0fc)

**Comment by RNarmatha** [comment]
Hi @Mohanapriya777 - 

Rajesh discussed the same with Manish last week, and instructions were given to check the certificate.

@bommalla - please confirm on this

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha @ma
Yes last week I discussed with manish,I suggested him to check the SSL check in the browser.

**Reply by Mohanapriya777** [reply]
Yes, but error was not resolved by that. The certificate and key files are matching but still we are getting error.

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
Please follow the below steps to updated the SSl certificate:

Step1 : Please move the validate SSL keys to cQube home directory.
Step2 : Please move SSL keys from cQube home directory to below mentioned path
            sudo docker exec -it nginx_app bash
            cd /etc/ssl/<vsk.ap.gov.in> your domain name
Step3 : After that please navigate the below mentioned path and modify the nginx template
            cd cqube-devops
            cd /opt/cqube/nginx
            vim default.conf
Step4: Please go inside default.conf and modify the below mentioned two lines.
           ssl_certificate /etc/ssl/<Domain_name>/sslcert.crt;
           ssl_certificate_key /etc/ssl/<Domain_Nmae>/sslcert.key;
Step5: After modifying the SSL path then we can restart the nginx container
           sudo docker restart nginx_app

**Reply by Mohanapriya777** [reply]
We tried the same and we still getting the same error.

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha Check port number 443 and 80 is enabled or not ?

**Reply by Mohanapriya777** [reply]
Let me check with the cloud team

**Reply by Mohanapriya777** [reply]
The ports are enabled

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 
I think please change the SSL keys names.keys contain numbers and pipe symbol is there in the above screenshot.need to change it

**Reply by Mohanapriya777** [reply]
Hi ,
The issue is resolved it is because of file name only. Thank you

**Comment by RNarmatha** [comment]
Hi @Mohanapriya777 

Thanks for the update
